src/bonds.cpp

```diff
@@ -224,9 +258,9 @@ void HarmonicTorsion::setEnergyFunction(const ParticleVector& particles) {
             const auto inverse_norm_bc = 1.0 / bc.norm();
             const auto angle = std::acos(ba.dot(bc) * inverse_norm_ba * inverse_norm_bc);
             const auto forcemagnitude = -2.0 * half_force_constant * (angle - equilibrium_angle);
```

Comment:
`force_magnitude`

Reply:
Fixed and rename other equivalent instance of `magnitude` to `force_magnitude`.
Fixed with f1feccc.